ztrouy/gondor-backend
# Separate Serializers From Views

### Purpose
To separate the serializers from the view files to prevent clutter and reduce duplicated code

### Files Changed
- Added calculated property "full_name" in `users.py`
- Created file for generalized serializers to embed in other serializers in new `embeds.py` in the `/serializers` directory
- Created file for model specific serializers in new `addresses.py`, `appointments.py`, `auths.py`, `clinicians.py`, `patient_data.py`, `patients.py`, and `users.py` in `/serializers` directory
- Create package file for serializers in new `__init__.py` in the `/serializers` directory
- Removed serializers, and updated endpoints to use imported serializers in `addresses.py`, `appointments.py`, `auth.py`, `records.py`, `states.py`, and `users.py` in `/views` directory

### To Test
Fetch, then run the following commands in the project root directory
```
pipenv shell
```
```
pipenv install
```
```
./seed_database.sh
```
Then confirm that all endpoints still work
